run-tracetesting-tests.sh

```diff
@@ -27,7 +27,7 @@ if [ "$RESTART" == "yes" ]; then
 fi
 docker compose -f docker-compose.yaml up -d --build --remove-orphans
 docker compose -f docker-compose.yaml -f local-config/docker-compose.testrunner.yaml build
-docker compose -f docker-compose.yaml -f local-config/docker-compose.testrunner.yaml run testrunner
+docker compose -f docker-compose.yaml -f local-config/docker-compose.testrunner.yaml run cypress
```

Comment:
```suggestion
docker compose -f docker-compose.yaml -f local-config/docker-compose.testrunner.yaml run tracetest
```

Reply:
i have 2 copies of this file. this one runs the testrunner. the other one run the cypress test

Reply:
Yes, this one should run tracetest, not cypress